### PR TITLE
Modify Gradle instructions to use a plugin instead of internal APIs.

### PR DESCRIPTION
**Pull request by aalmiray**
Modify Gradle instructions to use a plugin instead of internal APIs.
Fix for #6

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,18 +1,13 @@
-apply plugin: 'application'
+plugins {
+    id 'application'
+    id 'com.google.osdetector' version '1.6.0'
```
Comment (tiainen) [inline build.gradle]: There is a pending issue when applying the os detector plugin this way: https://github.com/google/osdetector-gradle-plugin/issues/15

You can either fallback to 1.4.0 or configure the plugin using `buildscript`.

Comment (abhinayagarwal) [reply build.gradle]: @aalmiray Any update on this?

Comment (aalmiray) [reply build.gradle]: Took care of it by reverting back to `buildscript` instead of `plugins` block

File: build.gradle [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
+    }
+}
+
 apply plugin: 'application'
+apply plugin: 'com.google.osdetector
```
Comment (tiainen) [inline build.gradle]: Missing a closing single quote.

**Pull request merged**